gateway/src/main/java/discord4j/gateway/intent/Intent.java

```diff
@@ -170,7 +182,45 @@ public enum Intent {
      *     <li>TYPING_START</li>
      * </ul>
      */
-    DIRECT_MESSAGE_TYPING(14);
+    DIRECT_MESSAGE_TYPING(14),
+    
+    /**
+     * MESSAGE_CONTENT does not represent individual events, but rather affects what data is present for
+     * events that could contain message content fields.
+     *
+     * @see <a href="https://discord.com/developers/docs/topics/gateway#message-content-intent">Message Contnet Intent</a>
+     */
+    MESSAGE_CONTENT(15),
```

Comment:
will setting this new intent  resolve message events appearing with empty content?